pkgs/development/python-modules/beanhub-cli/default.nix

```diff
@@ -0,0 +1,77 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, pythonOlder
+, pytz
+, pydantic
+, pydantic-core
+, pydantic-settings
+, pyyaml
+, jinja2
+, rich
+, fastapi
+, click
+, uvicorn
+, starlette-wtf
+, beancount-parser
+, beancount-black
+, beanhub-import
+, beanhub-forms
+, poetry-core
+, httpx
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "beanhub-cli";
+  version = "1.1.3";
+  format = "pyproject";
+
+  disabled = pythonOlder "3.10";
+
+  src = fetchFromGitHub {
+    owner = "LaunchPlatform";
+    repo = "beanhub-cli";
+    rev = "refs/tags/${version}";
+    hash = "sha256-vYBbaUVJAs+aIp6aQpJb62DEDxe/sQTzgOkjPq6ADoc=";
+  };
+
+  buildInputs = [
+    poetry-core
+  ];
+
+  propagatedBuildInputs = [
+    pytz
+    pydantic
+    pydantic-core
+    pydantic-settings
+    pyyaml
+    jinja2
+    rich
+    starlette-wtf
+    fastapi
+    click
+    uvicorn
+    beancount-parser
+    beancount-black
+    beanhub-import
+    beanhub-forms
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    httpx
+  ];
+
+  pythonImportsCheck = [
+    "beanhub_cli"
+  ];
+
+  meta = with lib; {
+    description = "Command line tools for BeanHub or Beancount users";
+    homepage = "https://github.com/LaunchPlatform/beanhub-cli/";
+    changelog = "https://github.com/LaunchPlatform/beanhub-cli/releases/tag/${version}";
+    license = with licenses; [ mit ];
+    maintainers = with maintainers; [ ambroisie ];
```

Comment:
I don't want to maintain this package, I don't make use of it.

Please add yourself to the maintainer list (in a commit named `maintains: add <handle>` beforein accordance with the [contributing docs](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#commit-conventions) :-).

Reply:
ah, sorry, didn't meant to throw these packages at your, forgot to add myself as an author 😅